packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -122,6 +122,14 @@ export const PARTITION_MAP: { [region: string]: Region } = {
   'us-isob-': { partition: Partition.UsIsoB, domainSuffix: 'sc2s.sgov.gov' },
 };
 
+export const CR_DEFAULT_RUNTIME_MAP: Record<Partition, string> = {
```

Comment:
I'm pretty sure this just got removed, and this PR is adding it back in.

Reply:
Compared the entire file and did the clean up.